src/components/common/Problem.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+	<div v-html="html" />
+</template>
+<script lang="ts">
+import { defineComponent, Ref, ref, watch } from 'vue';
+import axios from 'axios';
+
+import './mathjax-config';
+import 'mathjax-full/es5/tex-chtml.js';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+interface Window {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+  MathJax: any
+}
+
+export default defineComponent({
+	name: 'Problem',
+	props: {
+		file: {
+			type: String,
+			default: ''
+		}
+	},
+	setup(props){
+		const html: Ref<string> = ref('');
+		const _file: Ref<string> = ref(props.file);
+
+ 		async function loadProblem() {
+			const formData = new FormData();
+			formData.append('problemSeed', '1');
+			formData.append('sourceFilePath', _file.value);
+			formData.append('outputFormat', 'static');
```

Comment:
Not an issue here, but generally we will want to use `.set()` rather than `.append()` in that `set` will replace an existing key instead of `append` which will duplicate keys... it's just 'safer'

The exception is when WeBWorK problems use checkboxes, in which case it is expected that the same key might be repeated in the case that multiple boxes are checked. (I am 99% that is the only exception...)

Reply:
Yeah, set is probably safer than append here.  I quickly hacked this together for Peter this morning.  I didn't check the FormData api.